### DISCUSSION TITLE
Best long distance wmbus dongle? Metis II or iU891A-XL

### DISCUSSION THREAD

**Discussion by leifnel** [wmbusmeters/wmbusmeters#1430] [General]
I need to receive wmbus signals from quite a distance. 
Which receiver is the most sensitive?
Metis II have a sensivity of -106dB, the iU891A-XL -150dB according to datasheets.
That's sounds to me that the specifications are not compatible; that is too large a difference I think.

**Comment by Tpaydt** [comment]
i think the best is RTL-SRD Dongel with the Dipole Antenna

**Comment by weetmuts** [comment]
https://wmbusmeters.github.io/wmbusmeters-wiki/ANTENNA.html
